NapDatabaseExport/SVExportProviderBase.cs

```diff
@@ -24,7 +24,9 @@ public override void WriteColumnNames (string [] columns)
         public override void WriteRow (object [] values)
         {
             writer.WriteLine (string.Join (Delimiter,
-                values.Select (v => v != null && v != DBNull.Value ? GetCellValue (v.ToString ()) : string.Empty)));
+                values.Select (v => v != null && v != DBNull.Value
+                                ? GetCellValue (v is byte[] ? "0x" + BitConverter.ToString((byte[]) v).Replace("-", "") : v.ToString ())
```

Comment:
New line indentation should be 4 chars and space is required before braces and square brackets